packages/anvil/package.json

```diff
@@ -0,0 +1,5 @@
+{
+    "name": "anvil",
+    "private": true,
+    "version": "1.0.0"
```

Comment:
1.0.0 already?

Reply:
I think so. This is the version for the anvil image. There is not much to add, or not to be called 1.0